src/test/scala/is/hail/stats/SkatModelSuite.scala

```diff
@@ -0,0 +1,250 @@
+package is.hail.stats
```

Comment:
in tests, i think it's reasonable to keep especially helpful println, but with // in front so it doesn't fill the terminal and logs. We can check with Cotton.